tests/sqlparser_snowflake.rs

```diff
@@ -2846,3 +2846,32 @@ fn test_parse_show_columns_sql() {
     snowflake().verified_stmt("SHOW COLUMNS IN TABLE abc");
     snowflake().verified_stmt("SHOW COLUMNS LIKE '%xyz%' IN TABLE abc");
 }
+
+#[test]
+fn test_sf_double_dot_notation() {
+    snowflake().verified_stmt("SELECT * FROM db_name..table_name");
+    snowflake().verified_stmt("SELECT * FROM x, y..z JOIN a..b AS b ON x.id = b.id");
+}
+
+#[test]
+fn test_sf_double_dot_notation_wrong_position() {}
+
+#[test]
+fn test_parse_double_dot_notation_wrong_position() {
+    assert_eq!(
+        snowflake()
+            .parse_sql_statements("SELECT * FROM X.Y..")
+            .unwrap_err()
+            .to_string(),
+        "sql parser error: Expected: identifier, found: ."
+    );
+
+    assert_eq!(
+        // Ensure we don't parse leading token
+        snowflake()
+            .parse_sql_statements("SELECT * FROM .X.Y")
+            .unwrap_err()
+            .to_string(),
+        "sql parser error: Expected: identifier, found: ."
+    );
+}
```

Comment:
can we merge the methods into `test_sf_double_dot_notation()` so that the latter contains the different scenarios? thinking that would help keep the tests focused to the feature since the feature is limited enough in scope

Reply:
Sounds good